sdcm/wait.py

```diff
@@ -32,7 +32,11 @@ class WaitForTimeoutError(Exception):
     pass
 
 
-def wait_for(func, step=1, text=None, timeout=None, throw_exc=True, **kwargs):
+class ExitByEventError(Exception):
```

Comment:
We have dedicated module for it here: https://github.com/scylladb/scylla-cluster-tests/blob/master/sdcm/exceptions.py

Please, reuse it.

Reply:
@vponomaryov 

```
class WaitForTimeoutError(Exception):
    pass
```
 from the line 31, located in this file.
Should we move WaitForTimeoutError in this case as well?

Reply:
i moved both exceptions to sdcm.exceptions module